packages/css-data/package.json

```
@@ -26,7 +27,11 @@
   "peerDependencies": {
     "zod": "^3.19.1"
   },
-  "main": "lib/index.js",
+  "exports": {
+    "import": "./lib/index.js",
+    "require": "./lib/cjs/index.cjs",
```

To avoid dual package hazard in bundlers (although not in esbuild) you could also have a `module` condition here:
```suggestion
    "module": "./lib/index.js",
    "import": "./lib/index.js",
    "require": "./lib/cjs/index.cjs",
```

---

Looks like not a problem with remix right now. It always resolves modules. Ideally enable modules in designer as well.

---

Had to add old school "module" to make webpack4 in storybook work

---

> Looks like not a problem with remix right now. It always resolves modules.

That's specific to this repo - because you don't explicitly use `require` anywhere (so you always end up with the import condition). For potential external consumers, this wouldn't necessarily be always true.